website/server/models/task.js

```diff
@@ -25,6 +26,13 @@ export let TaskSchema = new Schema({
   type: {type: String, enum: tasksTypes, required: true, default: tasksTypes[0]},
   text: {type: String, required: true},
   notes: {type: String, default: ''},
+  shortName: {
+    type: String,
+    validate: [function validateShortName (val) {
+      return Boolean(this.userId) && !validator.isUUID(val) && val.match(SHORT_NAME_VALIDATION_REGEX);
+    },
+    'Task short names can only contain alphanumeric characters, underscores and dashses. They must be owned by a user.'],
```

Comment:
dashses -> dashes

"They must be owned by a user" will cause confusion.

A participant in an active challenge CAN give a short name to their own copies of the challenge tasks, yes? The message is saying that a challenge owner cannot set a short name on the tasks in the challenge itself? If so, it's probably best to state that explicitly.